# Single Keypoint 3D coordinates.

uint32 x
uint32 y
uint32 z
